packages/lib-classifier/src/plugins/drawingTools/models/marks/Ellipse/Ellipse.js

```diff
@@ -29,8 +29,8 @@ const EllipseModel = types
 
     deleteButtonPosition(scale) {
       const theta = DELETE_BUTTON_ANGLE * (Math.PI / 180)
-      const dx = ((self.rx + BUFFER) / scale) * Math.cos(theta)
-      const dy = ((self.ry + BUFFER) / scale) * Math.sin(theta)
+      const dx = (self.rx + BUFFER) * Math.cos(theta)
+      const dy = (self.ry + BUFFER) * Math.sin(theta)
```

Comment:
Why is this mark touched in this PR?

Reply:
There's a bug in the ellipse tool, where the delete button position changes when you zoom in and out. This fixes it.